src/main/java/com/training/web/pages/flipkart/ProductPage.java

```diff
@@ -1,13 +1,14 @@
 package com.training.web.pages.flipkart;
 
+import com.training.basepages.FlipkartBasePage;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class ProductPage {
+public class ProductPage extends FlipkartBasePage {
```

Comment:
remove driver & wait variables - as we do not need these as we are extending FlipkartBasePage